sdk/devcenter/azure-developer-devcenter/swagger/README.md

```diff
@@ -0,0 +1,20 @@
+## Generate autorest code
+
+```yaml
```

Comment:
Another nit, and something we should look into changing in all Swagger READMEs @weidongxu-microsoft and @srnagar 

I'd add a `use: <Autorest version>` to force Autorest version pinning:

```
use: '@autorest/java@4.1.5'
```

Reply:
At present DPG recommend using latest release https://github.com/Azure/azure-sdk-for-java/wiki/Protocol-Methods-Quickstart-with-AutoRest#follow-up-code-generation

One reason is, so far, DPG codegen is unlikely to break any API. And latest codegen usually have bug fixes.

Though it seems won't hurt if we put the codegen version in readme as well, since command line option should override markdown option.